public-ui/src/app/views/ehr-view/ehr-view.component.css

```diff
@@ -140,6 +140,41 @@ h5.secondary-display {
     font-size:1.3em;
 }
 
+.loading-dots {
```

Comment:
if this is the same as above. could this be moved into a common place ?

Reply:
Yep. I will add it to the loading component.